src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.Buffer.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System.Diagnostics;
```

Comment:
Feel free to commit the change as is, but it would be better if this file was called ...BitsBuffer.cs to match the type name.